<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QDialog" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>381</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>731</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>la gestion des clients</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>281</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ajout d'un client</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lCIN">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lnom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ladresse">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajo">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 italic 8pt &quot;Terminal&quot;;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lprenom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="background_2">
        <property name="geometry">
         <rect>
          <x>-60</x>
          <y>-10</y>
          <width>801</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/resource.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="tabrech">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>60</y>
          <width>251</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <zorder>background_2</zorder>
       <zorder>label</zorder>
       <zorder>label_2</zorder>
       <zorder>label_3</zorder>
       <zorder>lCIN</zorder>
       <zorder>lnom</zorder>
       <zorder>ladresse</zorder>
       <zorder>pushButton_ajo</zorder>
       <zorder>lprenom</zorder>
       <zorder>label_4</zorder>
       <zorder>tabrech</zorder>
       <zorder>lineEdit_6</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>pushButton_3</zorder>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>tableau des clients</string>
       </attribute>
       <widget class="QTableView" name="tableclient">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>431</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="background">
        <property name="geometry">
         <rect>
          <x>-80</x>
          <y>-10</y>
          <width>801</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/resource.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <zorder>background</zorder>
       <zorder>tableclient</zorder>
       <zorder>lineEdit_7</zorder>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>supprimer client</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_sup">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="background_3">
        <property name="geometry">
         <rect>
          <x>-60</x>
          <y>-10</y>
          <width>801</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/resource.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>background_3</zorder>
       <zorder>label_5</zorder>
       <zorder>lineEdit_5</zorder>
       <zorder>pushButton_sup</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>les cartes fidelit√©s</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/resource.jpg);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>ajout d'une carte</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero de carte</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>75</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Point de fidelit√©</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>26</x>
          <y>145</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>valider</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>affichage  des carte</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>711</width>
          <height>251</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/resource.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="tablecarte">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back To Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>12</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
